recipes/spacy-lookups-data/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "spacy-lookups-data" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spacy_lookups_data-{{ version }}.tar.gz
+  sha256: 7831260fa75d9a61b3dc16c0853662285e8403ba714042435611277ea9b94ca5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+  commands:
+    - python -m pytest --tb=native --pyargs spacy_lookups_data
+
+about:
+  home: https://github.com/explosion/spacy-lookups-data
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
It seems that the license is not being packaged with this project. You can package it manually if you want to